MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/BasicSymbolLayerActivity.java

```diff
@@ -55,117 +54,41 @@ protected void onCreate(Bundle savedInstanceState) {
-
-        // Add the marker image to map
-        style.addImage("my-marker-image", BitmapFactory.decodeResource(
-            BasicSymbolLayerActivity.this.getResources(), R.drawable.blue_marker_view));
-
-        // Adding an offset so that the bottom of the blue icon gets fixed to the coordinate, rather than the
-        // middle of the icon being fixed to the coordinate point.
-        style.addLayer(new SymbolLayer("marker-layer", "marker-source")
-            .withProperties(PropertyFactory.iconImage("my-marker-image"),
-                iconOffset(new Float[]{0f, -9f})));
-
-        // Add the selected marker source and layer
-        style.addSource(new GeoJsonSource("selected-marker"));
-
-        // Adding an offset so that the bottom of the blue icon gets fixed to the coordinate, rather than the
-        // middle of the icon being fixed to the coordinate point.
-        style.addLayer(new SymbolLayer("selected-marker-layer", "selected-marker")
-            .withProperties(PropertyFactory.iconImage("my-marker-image"),
-                iconOffset(new Float[]{0f, -9f})));
-
-        mapboxMap.addOnMapClickListener(BasicSymbolLayerActivity.this);
-      }
-    });
-  }
+    List<Feature> symbolLayerIconFeatureList = new ArrayList<>();
+    symbolLayerIconFeatureList.add(Feature.fromGeometry(
+      Point.fromLngLat(-57.225365, -33.213144)));
+    symbolLayerIconFeatureList.add(Feature.fromGeometry(
+      Point.fromLngLat(-54.14164, -33.981818)));
+    symbolLayerIconFeatureList.add(Feature.fromGeometry(
+      Point.fromLngLat(-56.990533, -30.583266)));
 
-  @Override
-  public boolean onMapClick(@NonNull LatLng point) {
-    Style style = mapboxMap.getStyle();
-    if (style != null) {
-      final SymbolLayer selectedMarkerSymbolLayer =
-          (SymbolLayer) style.getLayer("selected-marker-layer");
-
-      final PointF pixel = mapboxMap.getProjection().toScreenLocation(point);
-      List<Feature> features = mapboxMap.queryRenderedFeatures(pixel, "marker-layer");
-      List<Feature> selectedFeature = mapboxMap.queryRenderedFeatures(
-          pixel, "selected-marker-layer");
-
-      if (selectedFeature.size() > 0 && markerSelected) {
-        return false;
-      }
-
-      if (features.isEmpty()) {
-        if (markerSelected) {
-          deselectMarker(selectedMarkerSymbolLayer);
-        }
-        return false;
-      }
-
-      GeoJsonSource source = style.getSourceAs("selected-marker");
-      if (source != null) {
-        source.setGeoJson(FeatureCollection.fromFeatures(
-            new Feature[]{Feature.fromGeometry(features.get(0).geometry())}));
-      }
-
-      if (markerSelected) {
-        deselectMarker(selectedMarkerSymbolLayer);
-      }
-      if (features.size() > 0) {
-        selectMarker(selectedMarkerSymbolLayer);
-      }
-    }
-    return true;
-  }
+    mapboxMap.setStyle(new Style.Builder().fromUrl("mapbox://styles/mapbox/cjf4m44iw0uza2spb3q0a7s41")
```

Comment:
could you add some context why we are using this specific style?

Reply:
Using a style that isn't found in `Style` class  to just shake things up a little.

![Screen Shot 2019-05-13 at 1 02 59 PM](https://user-images.githubusercontent.com/4394910/57650357-783beb80-757f-11e9-896b-918095ab7658.png)